### DISCUSSION TITLE
Unable to Access the server from another device on the same network

### DISCUSSION THREAD

**Discussion by yashdubey132** [asreview/asreview#1717] [Q&A]
I have a remote computer which is intended to run ASReview. I fire up the server directly using `asreview lab --port 5555` and it opens in the browser. However, when I try to access the server from other computers on the same network, I see ERR_CONNECTION_REFUSED. I have allowed all requests through the firewall of the computer to the port where ASReview starts. Are there any further settings I might need to change to allow this request to go through?

**Comment by cskaandorp** [comment]
Hi there,

I need a bit more information about your setup. If you still need help, contact me at c.s.kaandorp@uu.nl

cheers

Cspr

**Reply by cskaandorp** [reply]
Update: @yashdubey132 tries runs a Windows cloud instance, but is not able to reach the server on the opened port. He will try Linux now with [asreview-server-stack](https://github.com/asreview/asreview-server-stack). The problem seems network related.
